desktop/src/main/java/bisq/desktop/main/offer/MutableOfferViewModel.java

```diff
@@ -732,16 +732,17 @@ public void onFocusOutMinAmountTextField(boolean oldValue, boolean newValue) {
             InputValidator.ValidationResult result = isBtcInputValid(minAmount.get());
             minAmountValidationResult.set(result);
             if (result.isValid) {
-                syncMinAmountWithAmount = dataModel.getMinAmount().getValue().equals(dataModel.getAmount().getValue());
+                Coin minAmount = dataModel.getMinAmount().get();
```

Comment:
Not sure if it is a good idea to name it the same as the class field. Makes it a little bit hard to read with this.minAmount/minAmount.

Reply:
Yes, good point. Fixed it.